bk-config/create_client_files.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+
+
+CONTAINER_CMD='docker run -i --rm --network beekeeper_default --name bk-config -v beekeeper-config_bk-secrets:/usr/lib/sage/:ro sagecontinuum/bk-config'
+
+
+
+if [ ! $# -eq 3 ]
+  then
+    echo "Usage: create_client_files.sh  HOST PORT MINUTES"
+    echo ""
```

Comment:
I would like to see some details here as to what `HOST`, `PORT` and `MINUTES` are used for.

for example:

HOST: the name of the beekeeper host the client is going to connect to
PORT: the port of the above mentioned host
MINUTES: the length of time (in minutes) the certificate is valid for

Reply:
done

Reply:
strange... now it should really be done.